HRESULT: 0x8007000E Out of memory

Hi, thanks for your hard work
[screenshot](https://imgur.com/a/cIpFL)
got r9 280 (2gb vmemory), 8 gb ram, 20gb+ on C, a lot on other partition
Refuses to work, dx9 still works as expected

Ok, with last Version
see the log with AMD R9 290 (last driver)

```

   ࡓharpDX.Result.CheckError()
   ࡓharpDX.Direct3D11.DeviceContext.MapSubresource(Resource resourceRef, Int32 subresource, MapMode mapType, MapFlags mapFlags)
   ࡈyperionScreenCap.DX11ScreenCapture.ManagedCapture()
   ࡈyperionScreenCap.DX11ScreenCapture.Capture()
   ࡈyperionScreenCap.MainForm.TransmitNextFrame()
   --- Fin de la trace de la pile d'exception interne ---
   ࡈyperionScreenCap.MainForm.TransmitNextFrame()
   ࡈyperionScreenCap.MainForm.StartCapture()
2017-10-19 13:20:25 [INFO] [4] HyperionScreenCap.MainForm.StartCapture:0 - Waiting before next screen capture attempt
2017-10-19 13:20:26 [ERROR] [4] HyperionScreenCap.MainForm.StartCapture:0 - Exception in screen capture attempt: 1
System.Exception: Error occured during capture: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Mꮯire insuffisante pour cette op곡tion.
 ---> SharpDX.SharpDXException: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Mꮯire insuffisante pour cette op곡tion.
```

with Nvidia 760 no problem with dx11

This is my log with 2.2 and [RX470]

> `bei HyperionScreenCap.DX11ScreenCapture.ManagedCapture()
>    bei HyperionScreenCap.DX11ScreenCapture.Capture()
>    bei HyperionScreenCap.MainForm.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.MainForm.TransmitNextFrame()
>    bei HyperionScreenCap.MainForm.StartCapture()
> 2017-10-20 20:45:20 [INFO] [6] HyperionScreenCap.MainForm.StartCapture:0 - Waiting before next screen capture attempt
> 2017-10-20 20:45:21 [ERROR] [6] HyperionScreenCap.MainForm.StartCapture:0 - Exception in screen capture attempt: 24
> System.Exception: Error occured during capture: HRESULT: [0x80070005], Module: [General], ApiCode: [E_ACCESSDENIED/General access denied error], Message: Zugriff verweigert
>  ---> SharpDX.SharpDXException: HRESULT: [0x80070005], Module: [General], ApiCode: [E_ACCESSDENIED/General access denied error], Message: Zugriff verweigert
> 
>    bei SharpDX.Result.CheckError()
>    bei SharpDX.DXGI.Output1.DuplicateOutput(ComObject deviceRef)
>    bei HyperionScreenCap.DX11ScreenCapture.InitDesktopDuplicator()
>    bei HyperionScreenCap.DX11ScreenCapture.Capture()
>    bei HyperionScreenCap.MainForm.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.MainForm.TransmitNextFrame()
>    bei HyperionScreenCap.MainForm.StartCapture()
> 2017-10-20 20:45:21 [INFO] [6] HyperionScreenCap.MainForm.StartCapture:0 - Waiting before next screen capture attempt
> 2017-10-20 20:45:22 [ERROR] [6] HyperionScreenCap.MainForm.StartCapture:0 - Exception in screen capture attempt: 1
> System.Exception: Error occured during capture: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
>  ---> SharpDX.SharpDXException: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
> `

```
> 2017-11-28 19:56:40 [INFO] [1] HyperionScreenCap.Program.Main:0 - Logger Initialized
> 2017-11-28 19:56:40 [INFO] [1] HyperionScreenCap.MainForm..ctor:0 - Instantiating MainForm
> 2017-11-28 19:56:40 [INFO] [1] HyperionScreenCap.SettingsManager.LoadSetttings:0 - Loading settings from user.config
> 2017-11-28 19:56:40 [INFO] [1] HyperionScreenCap.SettingsManager.LoadSetttings:0 - Loaded settings from user.config
> 2017-11-28 19:56:40 [INFO] [1] HyperionScreenCap.Helper.UpdateChecker.StartUpdateCheck:0 - Starting update check
> 2017-11-28 19:56:41 [INFO] [1] HyperionScreenCap.MainForm.Init:0 - Initialization requested with parameters reInit=False, forceOn=False
> 2017-11-28 19:56:41 [INFO] [1] HyperionScreenCap.MainForm.Init:0 - Initialization lock set
> 2017-11-28 19:56:41 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Processing toggle capture command: ON
> 2017-11-28 19:56:41 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Toggle capture lock set
> 2017-11-28 19:56:41 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Executing toggle capture command on a new Thread[IsBackground = True]
> 2017-11-28 19:56:41 [INFO] [1] HyperionScreenCap.MainForm.Init:0 - Initialization lock unset
> 2017-11-28 19:56:41 [INFO] [1] HyperionScreenCap.MainForm..ctor:0 - MainForm Instantiated
> 2017-11-28 19:56:44 [INFO] [4] HyperionScreenCap.MainForm.ExecuteToggleCaptureCommand:0 - Toggle capture lock unset
> 2017-11-28 19:56:51 [INFO] [1] HyperionScreenCap.Program.Main:0 - Logger Initialized
> 2017-11-28 19:56:51 [INFO] [1] HyperionScreenCap.MainForm..ctor:0 - Instantiating MainForm
> 2017-11-28 19:56:51 [INFO] [1] HyperionScreenCap.SettingsManager.LoadSetttings:0 - Loading settings from user.config
> 2017-11-28 19:56:51 [INFO] [1] HyperionScreenCap.SettingsManager.LoadSetttings:0 - Loaded settings from user.config
> 2017-11-28 19:56:51 [INFO] [1] HyperionScreenCap.Helper.UpdateChecker.StartUpdateCheck:0 - Starting update check
> 2017-11-28 19:56:52 [INFO] [1] HyperionScreenCap.MainForm.Init:0 - Initialization requested with parameters reInit=False, forceOn=False
> 2017-11-28 19:56:52 [INFO] [1] HyperionScreenCap.MainForm.Init:0 - Initialization lock set
> 2017-11-28 19:56:52 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Processing toggle capture command: ON
> 2017-11-28 19:56:52 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Toggle capture lock set
> 2017-11-28 19:56:52 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Executing toggle capture command on a new Thread[IsBackground = True]
> 2017-11-28 19:56:52 [INFO] [1] HyperionScreenCap.MainForm.Init:0 - Initialization lock unset
> 2017-11-28 19:56:52 [INFO] [1] HyperionScreenCap.MainForm..ctor:0 - MainForm Instantiated
> 2017-11-28 19:56:52 [INFO] [4] HyperionScreenCap.MainForm.EnableCapture:0 - Enabling 0 screen capture(s)
> 2017-11-28 19:56:52 [INFO] [4] HyperionScreenCap.Helper.HyperionTask.EnableCapture:0 - HyperionTask[ConfigurationId: 58b071]: Enabling screen capture
> 2017-11-28 19:56:52 [INFO] [4] HyperionScreenCap.MainForm.EnableCapture:0 - Enabled 1 screen capture(s)
> 2017-11-28 19:56:52 [INFO] [4] HyperionScreenCap.MainForm.ExecuteToggleCaptureCommand:0 - Toggle capture command ON completed
> 2017-11-28 19:56:52 [INFO] [4] HyperionScreenCap.MainForm.ExecuteToggleCaptureCommand:0 - Toggle capture lock unset
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.InitScreenCapture:0 - HyperionTask[ConfigurationId: 58b071]: Initializing screen capture
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.InitScreenCapture:0 - HyperionTask[ConfigurationId: 58b071]: Screen capture initialized
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.ConnectProtoClients:0 - HyperionTask[ConfigurationId: 58b071]: Connecting ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.ProtoClient.Dispose:0 - Disconnecting ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.ProtoClient.TryClearPriority:0 - Clearing Hyperion priority for ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.ProtoClient.TryClearPriority:0 - Hyperion priority cleared for ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.ProtoClient.Dispose:0 - ProtoClient[192.168.178.49:19445 (10)] disconnected
> 2017-11-28 19:56:52 [INFO] [5] HyperionScreenCap.ProtoClient.Connect:0 - ProtoClient[192.168.178.49:19445 (10)] Init lock set
> 2017-11-28 19:56:54 [INFO] [1] HyperionScreenCap.MainForm.ShowSetupFrom:0 - Loading SetupForm
> 2017-11-28 19:56:54 [INFO] [1] HyperionScreenCap.SetupForm..ctor:0 - Instantiating SetupForm
> 2017-11-28 19:56:54 [INFO] [1] HyperionScreenCap.SetupForm.LoadSettings:0 - Loading settings using SettingsManager
> 2017-11-28 19:56:54 [INFO] [1] HyperionScreenCap.SettingsManager.LoadSetttings:0 - Loading settings from user.config
> 2017-11-28 19:56:54 [INFO] [1] HyperionScreenCap.SettingsManager.LoadSetttings:0 - Loaded settings from user.config
> 2017-11-28 19:56:54 [INFO] [1] HyperionScreenCap.SetupForm.LoadSettings:0 - Finished loading settings using SettingsManager
> 2017-11-28 19:56:54 [INFO] [1] HyperionScreenCap.SetupForm..ctor:0 - SetupForm Instantiated
> 2017-11-28 19:56:56 [INFO] [1] HyperionScreenCap.MainForm.TrayIcon_DoubleClick:0 - Tray icon double clicked
> 2017-11-28 19:56:56 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Processing toggle capture command: OFF
> 2017-11-28 19:56:56 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Toggle capture lock set
> 2017-11-28 19:56:56 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Executing toggle capture command on a new Thread[IsBackground = True]
> 2017-11-28 19:56:56 [INFO] [4] HyperionScreenCap.MainForm.DisableCapture:0 - Disabling 1 screen capture(s)
> 2017-11-28 19:56:56 [INFO] [4] HyperionScreenCap.Helper.HyperionTask.DisableCapture:0 - HyperionTask[ConfigurationId: 58b071]: Disabling screen capture
> 2017-11-28 19:56:58 [INFO] [4] HyperionScreenCap.MainForm.DisableCapture:0 - Disabled 1 screen capture(s)
> 2017-11-28 19:56:58 [INFO] [4] HyperionScreenCap.MainForm.ExecuteToggleCaptureCommand:0 - Toggle capture command OFF completed
> 2017-11-28 19:56:58 [INFO] [4] HyperionScreenCap.MainForm.ExecuteToggleCaptureCommand:0 - Toggle capture lock unset
> 2017-11-28 19:57:01 [INFO] [5] HyperionScreenCap.ProtoClient.Connect:0 - ProtoClient[192.168.178.49:19445 (10)] Init lock unset
> 2017-11-28 19:57:01 [ERROR] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 1
> System.Exception: Failed to connect to Hyperion server using ProtoClient[192.168.178.49:19445 (10)] ---> System.Net.Sockets.SocketException: Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte 192.168.178.49:19445
>    bei System.Net.Sockets.TcpClient.Connect(String hostname, Int32 port)
>    bei HyperionScreenCap.ProtoClient.Connect()
>    bei HyperionScreenCap.Helper.HyperionTask.ConnectProtoClients()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.Helper.HyperionTask.ConnectProtoClients()
>    bei HyperionScreenCap.Helper.HyperionTask.StartCapture()
> 2017-11-28 19:57:01 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Waiting before next screen capture attempt
> 2017-11-28 19:57:02 [INFO] [5] HyperionScreenCap.ProtoClient.Dispose:0 - Disconnecting ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:57:02 [INFO] [5] HyperionScreenCap.ProtoClient.TryClearPriority:0 - Clearing Hyperion priority for ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:57:02 [INFO] [5] HyperionScreenCap.ProtoClient.TryClearPriority:0 - Hyperion priority cleared for ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:57:02 [INFO] [5] HyperionScreenCap.ProtoClient.Dispose:0 - ProtoClient[192.168.178.49:19445 (10)] disconnected
> 2017-11-28 19:57:02 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.TryStartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Screen Capture finished
> 2017-11-28 19:58:56 [INFO] [1] HyperionScreenCap.MainForm.TrayIcon_DoubleClick:0 - Tray icon double clicked
> 2017-11-28 19:58:56 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Processing toggle capture command: ON
> 2017-11-28 19:58:56 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Toggle capture lock set
> 2017-11-28 19:58:56 [INFO] [1] HyperionScreenCap.MainForm.ToggleCapture:0 - Executing toggle capture command on a new Thread[IsBackground = True]
> 2017-11-28 19:58:56 [INFO] [7] HyperionScreenCap.MainForm.EnableCapture:0 - Enabling 1 screen capture(s)
> 2017-11-28 19:58:56 [INFO] [7] HyperionScreenCap.Helper.HyperionTask.EnableCapture:0 - HyperionTask[ConfigurationId: 58b071]: Enabling screen capture
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.Helper.HyperionTask.InitScreenCapture:0 - HyperionTask[ConfigurationId: 58b071]: Initializing screen capture
> 2017-11-28 19:58:56 [INFO] [7] HyperionScreenCap.MainForm.EnableCapture:0 - Enabled 1 screen capture(s)
> 2017-11-28 19:58:56 [INFO] [7] HyperionScreenCap.MainForm.ExecuteToggleCaptureCommand:0 - Toggle capture command ON completed
> 2017-11-28 19:58:56 [INFO] [7] HyperionScreenCap.MainForm.ExecuteToggleCaptureCommand:0 - Toggle capture lock unset
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.Helper.HyperionTask.InitScreenCapture:0 - HyperionTask[ConfigurationId: 58b071]: Screen capture initialized
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.Helper.HyperionTask.ConnectProtoClients:0 - HyperionTask[ConfigurationId: 58b071]: Connecting ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.ProtoClient.Dispose:0 - Disconnecting ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.ProtoClient.TryClearPriority:0 - Clearing Hyperion priority for ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.ProtoClient.TryClearPriority:0 - Hyperion priority cleared for ProtoClient[192.168.178.49:19445 (10)]
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.ProtoClient.Dispose:0 - ProtoClient[192.168.178.49:19445 (10)] disconnected
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.ProtoClient.Connect:0 - ProtoClient[192.168.178.49:19445 (10)] Init lock set
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.ProtoClient.Connect:0 - ProtoClient[192.168.178.49:19445 (10)] Init lock unset
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.Helper.HyperionTask.ConnectProtoClients:0 - HyperionTask[ConfigurationId: 58b071]: ProtoClient[192.168.178.49:19445 (10)] connected
> 2017-11-28 19:58:56 [INFO] [8] HyperionScreenCap.NotificationUtils.Info:0 - Info notification: Connected to Hyperion server using ProtoClient[192.168.178.49:19445 (10)]!
> 2017-11-28 19:58:57 [ERROR] [8] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 1
> System.Exception: Error occured while sending image to server: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
>  ---> SharpDX.SharpDXException: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
> 
>    bei SharpDX.Result.CheckError()
>    bei SharpDX.Direct3D11.DeviceContext.MapSubresource(Resource resourceRef, Int32 subresource, MapMode mapType, MapFlags mapFlags)
>    bei HyperionScreenCap.DX11ScreenCapture.ManagedCapture()
>    bei HyperionScreenCap.DX11ScreenCapture.Capture()
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    bei HyperionScreenCap.Helper.HyperionTask.StartCapture()
> 2017-11-28 19:58:57 [INFO] [8] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Waiting before next screen capture attempt
> 2017-11-28 19:58:58 [ERROR] [8] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 2
> System.Exception: Error occured while sending image to server: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
>  ---> SharpDX.SharpDXException: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
> 
>    bei SharpDX.Result.CheckError()
>    bei SharpDX.Direct3D11.DeviceContext.MapSubresource(Resource resourceRef, Int32 subresource, MapMode mapType, MapFlags mapFlags)
>    bei HyperionScreenCap.DX11ScreenCapture.ManagedCapture()
>    bei HyperionScreenCap.DX11ScreenCapture.Capture()
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    bei HyperionScreenCap.Helper.HyperionTask.StartCapture()
> 2017-11-28 19:58:58 [INFO] [8] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Waiting before next screen capture attempt
> 2017-11-28 19:58:59 [ERROR] [8] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 1
> System.Exception: Error occured while sending image to server: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
>  ---> SharpDX.SharpDXException: HRESULT: [0x8007000E], Module: [General], ApiCode: [E_OUTOFMEMORY/Out of memory], Message: Für diesen Vorgang ist nicht genügend Speicher verfügbar.
```
ANd also while starting up I get this. Software starts, and immediately quits (I start as admin). This is with DX9
```

> 2017-11-29 20:00:30 [ERROR] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 1
> System.Exception: Error occured while sending image to server: D3DERR_INVALIDCALL: Invalid call (-2005530516) ---> SlimDX.Direct3D9.Direct3D9Exception: D3DERR_INVALIDCALL: Invalid call (-2005530516)
>    bei SlimDX.Result.Throw[T](Object dataKey, Object dataValue)
>    bei SlimDX.Result.Record[T](Int32 hr, Boolean failed, Object dataKey, Object dataValue)
>    bei SlimDX.Direct3D9.Device.GetFrontBufferData(Int32 swapChain, Surface destinationSurface)
>    bei HyperionScreenCap.DX9ScreenCapture.GetCaptureSurface()
>    bei HyperionScreenCap.DX9ScreenCapture.Capture()
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    bei HyperionScreenCap.Helper.HyperionTask.StartCapture()
> 2017-11-29 20:00:30 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Waiting before next screen capture attempt
> 2017-11-29 20:00:31 [ERROR] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 2
> System.Exception: Error occured while sending image to server: D3DERR_INVALIDCALL: Invalid call (-2005530516) ---> SlimDX.Direct3D9.Direct3D9Exception: D3DERR_INVALIDCALL: Invalid call (-2005530516)
>    bei SlimDX.Result.Throw[T](Object dataKey, Object dataValue)
>    bei SlimDX.Result.Record[T](Int32 hr, Boolean failed, Object dataKey, Object dataValue)
>    bei SlimDX.Direct3D9.Device.GetFrontBufferData(Int32 swapChain, Surface destinationSurface)
>    bei HyperionScreenCap.DX9ScreenCapture.GetCaptureSurface()
>    bei HyperionScreenCap.DX9ScreenCapture.Capture()
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    bei HyperionScreenCap.Helper.HyperionTask.StartCapture()
> 2017-11-29 20:00:31 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Waiting before next screen capture attempt
> 2017-11-29 20:00:57 [ERROR] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 1
> System.Exception: Error occured while sending image to server: D3DERR_INVALIDCALL: Invalid call (-2005530516) ---> SlimDX.Direct3D9.Direct3D9Exception: D3DERR_INVALIDCALL: Invalid call (-2005530516)
>    bei SlimDX.Result.Throw[T](Object dataKey, Object dataValue)
>    bei SlimDX.Result.Record[T](Int32 hr, Boolean failed, Object dataKey, Object dataValue)
>    bei SlimDX.Direct3D9.Device.GetFrontBufferData(Int32 swapChain, Surface destinationSurface)
>    bei HyperionScreenCap.DX9ScreenCapture.GetCaptureSurface()
>    bei HyperionScreenCap.DX9ScreenCapture.Capture()
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    bei HyperionScreenCap.Helper.HyperionTask.StartCapture()
> 2017-11-29 20:00:57 [INFO] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Waiting before next screen capture attempt
> 2017-11-29 20:00:58 [ERROR] [5] HyperionScreenCap.Helper.HyperionTask.StartCapture:0 - HyperionTask[ConfigurationId: 58b071]: Exception in screen capture attempt: 2
> System.Exception: Error occured while sending image to server: D3DERR_INVALIDCALL: Invalid call (-2005530516) ---> SlimDX.Direct3D9.Direct3D9Exception: D3DERR_INVALIDCALL: Invalid call (-2005530516)
>    bei SlimDX.Result.Throw[T](Object dataKey, Object dataValue)
>    bei SlimDX.Result.Record[T](Int32 hr, Boolean failed, Object dataKey, Object dataValue)
>    bei SlimDX.Direct3D9.Device.GetFrontBufferData(Int32 swapChain, Surface destinationSurface)
>    bei HyperionScreenCap.DX9ScreenCapture.GetCaptureSurface()
>    bei HyperionScreenCap.DX9ScreenCapture.Capture()
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    --- Ende der internen Ausnahmestapelüberwachung ---
>    bei HyperionScreenCap.Helper.HyperionTask.TransmitNextFrame()
>    bei HyperionScreenCap.Helper.HyperionTask.StartCapture()

```

--- a/HyperionScreenCap/Capture/Dx11ScreenCapture.cs
+++ b/HyperionScreenCap/Capture/Dx11ScreenCapture.cs
@@ -225,6 +225,7 @@ namespace HyperionScreenCap
             finally
             {
                 screenResource?.Dispose();
+                _device.ImmediateContext.UnmapSubresource(_stagingTexture, 0);
                 // Ignore DXGI_ERROR_INVALID_CALL, DXGI_ERROR_ACCESS_LOST errors since capture is already complete
                 try { _duplicatedOutput.ReleaseFrame(); } catch { }
             }

